platforms/allay/src/main/java/org/allaymc/terra/allay/generator/AllayGeneratorWrapper.java

```diff
@@ -0,0 +1,160 @@
+package org.allaymc.terra.allay.generator;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.allaymc.api.utils.AllayStringUtils;
+import org.allaymc.api.world.biome.BiomeType;
+import org.allaymc.api.world.generator.WorldGenerator;
+import org.allaymc.api.world.generator.context.NoiseContext;
+import org.allaymc.api.world.generator.context.PopulateContext;
+import org.allaymc.api.world.generator.function.Noiser;
+import org.allaymc.api.world.generator.function.Populator;
+import org.allaymc.terra.allay.TerraAllayPlugin;
+import org.allaymc.terra.allay.delegate.AllayProtoChunk;
+import org.allaymc.terra.allay.delegate.AllayProtoWorld;
+import org.allaymc.terra.allay.delegate.AllayServerWorld;
+
+import java.util.Locale;
+
+import com.dfsek.terra.api.config.ConfigPack;
+import com.dfsek.terra.api.world.biome.generation.BiomeProvider;
+import com.dfsek.terra.api.world.chunk.generation.ChunkGenerator;
+import com.dfsek.terra.api.world.chunk.generation.util.GeneratorWrapper;
+import com.dfsek.terra.api.world.info.WorldProperties;
+
+/**
+ * @author daoge_cmd
+ */
+@Slf4j
+public class AllayGeneratorWrapper implements GeneratorWrapper {
+
+    protected static final String DEFAULT_PACK_NAME = "overworld";
```

Comment:
For consistency with other platforms I'd prefer if Terra was disabled if the pack is not set